.pre-commit-config.yaml

```diff
@@ -418,6 +418,7 @@ repos:
         name: Check Shell scripts syntax correctness
         language: docker_image
         entry: koalaman/shellcheck:stable -x -a
+        exclude: activate$
```

Comment:
Is this file excluded by default because it is not added to the repository and is located in the `/files` directory?

Reply:
Ah. leftover. I remve it